### ISSUE TITLE
Incorrect reporting of colour contrast in Accessibility category

### ISSUE THREAD

**Issue by jcush** [GoogleChrome/lighthouse#14275]
Today I've been running Lighthouse (via Chrome's DevTools) on a website and the Accessibility score is missing a few points (97/100)

When I looked to see what is causing this, it's mostly a list of elements under the item:
_"Background and foreground colors do not have a sufficient contrast ratio."_

However, I believe this is reporting incorrectly after checking three of the combinations reported:
1. Foreground: #**00147a** / Background: **#ffffff** / Contrast:  15.2:1
2. Foreground: #**00147a** / Background: **#99e596** / Contrast: 10.13:1
3. Foreground: #**000038** / Background: **#ffffff** / Contrast: 13.24:1

I'm running Lighthouse in InCognito mode and the smallest font size for which any of these supposed errors occurs is 16 px / 12pt.

Having checked one of the colours in DevTools, I see that no contrast information can be provided so it's possible it's defaulting to a failure?

<img width="234" alt="image" src="https://user-images.githubusercontent.com/10184859/183880219-a71e28dd-5e2c-47fa-8508-2195e4fee03b.png">

Is this something I'm doing wrong somewhere or an issue with Lighthouse/DevTools?

Thanks

**Comment by adamraine** [comment]
Are you able to share a URL where this is happening. In an isolated environment, all of those FG/BG combinations should pass the audit.

For reference, this is the site I was testing on:
```html
<style>
  #first {
    color: #00147a;
    background-color: #ffffff;
  }
  #second {
    color: #00147a;
    background-color: #99e596;
  }
  #thid {
    color: #000038;
    background-color: #ffffff;
  }
</style>

<h1 id="first">Test 1</h1>
<h1 id="second">Test 2</h1>
<h1 id="third">Test 3</h1>
```

**Comment by jcush** [comment]
Hi,

Unfortunately I don't, however I can DM over a login to the staging site which is password-protected if this works.

The difference I can see above is you've explicitly defined the background colour on the same element. In this case the foreground colour of some of the elements in question are shown against the body's background colour (or a parent element).

Thanks

**Comment by adamraine** [comment]
> In my case the foreground colour of some of the elements in question are shown against the body's background colour (or a parent element).

The element containing the background color shouldn't affect the results here.

If my example doesn't reflect your page, can you share an HTML document that does?

**Comment by jcush** [comment]
Sure, when I get a moment I'll try and put one together

**Comment by maxhoffmann** [comment]
I have the same issue. It happens on my personal blog at [max.hn](https://max.hn). The footer links are dark grey text on a white background, yet they fail to pass the color contrast page. This is odd as clicking on the color in the inspector shows, that the color contrast is sufficient and passes both AA and AAA guidelines. (see screenshots) I’ve already tried changing the color, but it didn’t help.

<img width="944" alt="Screenshot 2023-05-26 at 10 55 28" src="https://github.com/GoogleChrome/lighthouse/assets/1227441/51776e6b-143e-4188-81ee-ba98d77ad11c">
<img width="990" alt="Screenshot 2023-05-26 at 10 55 13" src="https://github.com/GoogleChrome/lighthouse/assets/1227441/22813117-8c58-4a88-a470-9001c549a73c">
<img width="1042" alt="Screenshot 2023-05-26 at 10 54 51" src="https://github.com/GoogleChrome/lighthouse/assets/1227441/089f7817-42d8-4a62-9852-58043372a473">
